adapter_author_deps.bzl

```diff
@@ -18,6 +19,11 @@ def mixer_adapter_repositories():
         actual = "@com_github_google_protobuf//:protoc",
     )
 
+    native.bind(
```

Comment:
should we remove the `native.bind` rule for `protoc` too (now that pubref uses `procotol_compiler`)?

Reply:
cleanup. I would prefer to get this PR in and not take any chances for another e2e test run that might fail on me. I will send a new PR after this to clean this up and we can git it in after 0.2.